### DISCUSSION TITLE
Flet app v1.5

### DISCUSSION THREAD

**Discussion by algotrading-py** [flet-dev/flet#2849] [Q&A]
### Question

Hi everyone, please help.  I can't install https://play.google.com/store/apps/details?id=com.appveyor.flet
because it is not available on google play in my region.
Help me to download the distribution for self-installation. Where can I get it? VPN and different ways do not help to bypass the restriction.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
You can subscribe to this issue: https://github.com/flet-dev/flet/issues/2537
